File: inventario_visual/Models/Producto.vb
Namespace inventario_visual
    Public Class Producto
        Public Property Id As Long
        Public Property Sku As String
        Public Property Nombre As String
        Public Property Descripcion As String
        Public Property CategoriaId As Long?
        Public Property PrecioVenta As Double
        Public Property PrecioCompra As Double
        Public Property StockActual As Double
        Public Property StockMinimo As Double
        Public Property State As Boolean = True
        Public Property Imagen As String
        Public Property CategoriaNombre As String
    End Class
End Namespace

File: inventario_python/models/producto.py
class Producto:
    def __init__(self):
        self.id = 0
        self.categoria_id = None
        self.sku = ""
        self.nombre = ""
        self.descripcion = ""
        self.precio_venta = 0.0
        self.costo_promedio = 0.0
        self.stock_minimo = 0.0
        self.stock_actual = 0.0
        self.state = True
        self.imagen = None
        self.categoria_nombre = ""

    def __str__(self):
        return self.nombre
